beacon-chain/cache/shuffled_indices.go

```diff
@@ -59,10 +61,11 @@ func NewShuffledIndicesCache() *ShuffledIndicesCache {
 
 // ShuffledIndicesBySeed fetches ShuffledIndicesBySeed by epoch and seed. Returns true with a
```

Comment:
Does this comment need updating? Says `by epoch and seed` but index and seed are being passed as parameters.

Reply:
That makes sense to me!